src/shared/errors/throw-if-icon-spec-missing.ts

```diff
@@ -0,0 +1,12 @@
+import { IconLookup } from '@fortawesome/fontawesome';
+
+/**
+ * Throws if icon spec missing.
+ * @param {IconLookup} iconSpec
+ */
+export const faThrowIfIconSpecMissing = (iconSpec: IconLookup) => {
```

Comment:
Similarly, how about we call this `faWarnIfIconSpecMissing`, since we're not actually `throw`-ing anything?